lib/core/Canvas.js

```diff
@@ -464,6 +470,30 @@ Canvas.prototype.findPlane = function(element) {
   });
 };
 
+
+/**
+ * Removes the given plane
+ *
+ * @param {string|djs.model.Base} plane
+ */
+Canvas.prototype.removePlane = function(plane) {
+  if (typeof plane === 'string') {
+    plane = this.getPlane(plane);
+  }
+
+  if (!plane) {
+    throw new Error('must specify a plane');
+  }
+
+  // remove root element
+  this.setRootElementForPlane(null, plane, true);
```

Comment:
What if plane === activePlane? Any precautions we'd like to set?

Reply:
Also, this particular part needs further clarification. What is happening here (and why)?

Reply:
I was thinking about it with the renaming use-case in mind. There, we would want to reuse the existing root element on the newly created plane, and therefore need to remove it first.

I think deleting a plane should work like [`canvas._clear()`](https://github.com/bpmn-io/diagram-js/blob/91626d03b756573951c687b72ff21b81c0a3db80/lib/core/Canvas.js#L229-L252) on a plane level. 
So in case it is the active plane, it should set the active plane to `null`. The integrating library takes care of setting either a new one, or it resets to the default plane. (this is still missing)

Reply:
You are raising an important point here: How is this supposed to be used (i.e. to rename vs. add and remove)? What responsibility lies within the Canvas and what within the integrating library?

If we got clarity on this, please consider adding an integration test to verify this particular scenario in a more complex setup (i.e. renaming a plane, with contents, as the `id` changes or whatever).

I'm fine to keep the API low-level, if it is clear how it is supposed to be used to accomplish certain tasks, i.e. renaming.

Reply:
Agreed, I'll get on that :+1: